### PR TITLE
zh-translation: update all Istio's security news.

### PR DESCRIPTION
**Pull request by elkan1788**
Upgrade the Chinese translation in path:  content/zh/news/security/*

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/news/security/istio-security-2021-001/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+* __[CVE-2021-21378](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-21378)__:
+  JWT 身份验证绕过未知的颁发者令牌
+    * CVSS Score: 8.2 [AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:L/A:N](https://www.first.org/cvss/calculator/3.0#CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:L/A:N)
+
+如果仅将 `RequestAuthentication` 用于 JWT 验证，则您会受到此漏洞的影响。
+
+如果您同时使用 `RequestAuthentication` 和 `AuthorizationPolicy` 进行 JWT 验证，则您不受此漏洞的影响。
+
+{{< warning >}}
+请注意，`RequestAuthentication` 用于定义应接受的发行者列表。 它不会拒绝没有 JWT 令牌请求。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2021-001/index.md:27]:
```suggestion
请注意，`RequestAuthentication` 用于定义应接受的发行者列表。 它不会拒绝没有 JWT 令牌请求。
```

File: content/zh/news/security/istio-security-2020-008/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Istio 用户通过以下方式暴露此漏洞:
+
+* 通过 [Envoy 筛选器](/docs/reference/config/networking/envoy-filter/)直接使用 Envoy 的`verify_subject_alt_name`和`match_subject_alt_names`配置。
+
+* 使用Istio 的[ `subjectAltNames` 字段在目标规则与客户端 TLS 设置](/docs/reference/config/networking/destination-rule/#ClientTLSSettings).  具有包含 `nested.subdomain.example.com` 的 `subjectAltNames` 字段的目标规则错误地接受来自主机名称（SAN）为 `* .example.com` 的上游层证书。 相反，应该存在一个 `*.subdomain.example.com` 或 `nested.subdomain.example.com` 的 SAN。
+
+* 使用 Istio 的 [服务条目中的 `subjectAltNames`](/docs/reference/config/networking/service-entry/)。 带有 ` subjectAltNames` 字段的值类似于 `nested.subdomain.example.com` 的服务条目错误地接受来自 SAN 为 `*.example.com` 的上游层证书。
+
+Istio CA（以前称为 Citadel）不使用 DNS 通配符 SAN 颁发证书。 该漏洞仅影响验证外部颁发证书的配置。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-008/index.md:30]:
```suggestion
Istio CA（以前称为 Citadel）不使用 DNS 通配符 SAN 颁发证书。 该漏洞仅影响验证外部颁发证书的配置。
```

File: content/zh/news/security/istio-security-2020-008/index.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+当验证 TLS 证书时，Envoy 错误地允许将通配符 DNS 使用者主机名称应用于多个子域。例如，在 SAN 为 `*.example.com` 通配的情况下，Envoy 错误地允许使用 `nested.subdomain.example.com`，而它只允许使用 `subdomain.example.com`。
+    * CVSS Score: 6.6 [AV:N/AC:H/PR:H/UI:N/S:C/C:H/I:L/A:N/E:F/RL:O/RC:C](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:H/PR:H/UI:N/S:C/C:H/I:L/A:N/E:F/RL:O/RC:C&version=3.1)
+
+Istio 用户通过以下方式暴露此漏洞:
+
+* 通过 [Envoy 筛选器](/docs/reference/config/networking/envoy-filter/)直接使用 Envoy 的`verify_subject_alt_name`和`match_subject_alt_names`配置。
+
+* 使用Istio 的[ `subjectAltNames` 字段在目标规则与客户端 TLS 设置](/docs/reference/config/networking/destination-rule/#ClientTLSSettings).  具有包含 `nested.subdomain.example.com` 的 `subjectAltNames` 字段的目标规则错误地接受来自主机名称（SAN）为 `* .example.com` 的上游层证书。 相反，应该存在一个 `*.subdomain.example.com` 或 `nested.subdomain.example.com` 的 SAN。
+
+* 使用 Istio 的 [服务条目中的 `subjectAltNames`](/docs/reference/config/networking/service-entry/)。 带有 ` subjectAltNames` 字段的值类似于 `nested.subdomain.example.com` 的服务条目错误地接受来自 SAN 为 `*.example.com` 的上游层证书。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-008/index.md:28]:
```suggestion
* 使用 Istio 的 [服务条目中的 `subjectAltNames`](/zh/docs/reference/config/networking/service-entry/)。 带有 `subjectAltNames` 字段的值类似于 `nested.subdomain.example.com` 的服务条目错误地接受来自 SAN 为 `*.example.com` 的上游层证书。
```

File: content/zh/news/security/istio-security-2020-008/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+* __[`CVE-2020-15104`](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-15104)__:
+当验证 TLS 证书时，Envoy 错误地允许将通配符 DNS 使用者主机名称应用于多个子域。例如，在 SAN 为 `*.example.com` 通配的情况下，Envoy 错误地允许使用 `nested.subdomain.example.com`，而它只允许使用 `subdomain.example.com`。
+    * CVSS Score: 6.6 [AV:N/AC:H/PR:H/UI:N/S:C/C:H/I:L/A:N/E:F/RL:O/RC:C](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:H/PR:H/UI:N/S:C/C:H/I:L/A:N/E:F/RL:O/RC:C&version=3.1)
+
+Istio 用户通过以下方式暴露此漏洞:
+
+* 通过 [Envoy 筛选器](/docs/reference/config/networking/envoy-filter/)直接使用 Envoy 的`verify_subject_alt_name`和`match_subject_alt_names`配置。
+
+* 使用Istio 的[ `subjectAltNames` 字段在目标规则与客户端 TLS 设置](/docs/reference/config/networking/destination-rule/#ClientTLSSettings).  具有包含 `nested.subdomain.example.com` 的 `subjectAltNames` 字段的目标规则错误地接受来自主机名称（SAN）为 `* .example.com` 的上游层证书。 相反，应该存在一个 `*.subdomain.example.com` 或 `nested.subdomain.example.com` 的 SAN。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-008/index.md:26]:
```suggestion
* 使用Istio 的[`subjectAltNames` 字段在目标规则与客户端 TLS 设置](/zh/docs/reference/config/networking/destination-rule/#ClientTLSSettings)。具有包含 `nested.subdomain.example.com`  的  `subjectAltNames`  字段的目标规则错误地接受来自主机名称（SAN）为  `* .example.com`  的上游层证书。 相反，应该存在一个  `*.subdomain.example.com`  或 `nested.subdomain.example.com`  的  SAN。
```

File: content/zh/news/security/istio-security-2020-009/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+spec:
+ action: DENY
+ rules:
+ - from:
+   - source:
+       principals:
+       - */ns/ns1/sa/foo # indicating any trust domain, ns1 namespace, foo svc account
+{{< /text >}}
+
+Istio 将 Principal (和 `source.principal ` )字段转换为 Envoy 级别的字符串匹配。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-009/index.md:39]:
```suggestion
Istio 将 Principal (和 `source.principal` ) 字段转换为 Envoy 级别的字符串匹配。
```

File: content/zh/news/security/istio-security-2020-009/index.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+{{< text yaml >}}
+stringMatch:
+  suffix: spiffe:///ns/ns1/sa/foo
+{{< /text >}}
+
+它将不匹配任何合法的调用者，因为它错误的包含了 `spiffe:// ` 字符串。正确的字符串匹配应该是：
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-009/index.md:46]:
```suggestion
它将不匹配任何合法的调用者，因为它错误的包含了 `spiffe://` 字符串。正确的字符串匹配应该是：
```

File: content/zh/news/security/istio-security-2020-007/index.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+* 对于 1.5.x 部署： 请升级至  [Istio 1.5.7](/news/releases/1.5.x/announcing-1.5.7) 或更高的版本。
+* 对于 1.6.x 部署： 请升级至  [Istio 1.6.4](/news/releases/1.6.x/announcing-1.6.4) 或更高的版本。
+
+{{< warning >}}
+您必须采取以下附加步骤来防范 CVE-2020-8663。
+{{< /warning >}}
+
+
+通过添加[下游连接](https://www.envoyproxy.io/docs/envoy/v1.14.3/configuration/operations/overload_manager/overload_manager#limiting-active-connections)的配置限制，在 Envoy 解决 CVE-2020-8663。 必须配置此限制以防范此漏洞。 在 Ingress 网关上执行如下步骤来配置限制。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-007/index.md:44]:
```suggestion
通过添加[下游连接](https://www.envoyproxy.io/docs/envoy/v1.14.3/configuration/operations/overload_manager/overload_manager#limiting-active-connections)的配置限制，在 Envoy 解决 CVE-2020-8663。 必须配置此限制以防范此漏洞。在 Ingress 网关上执行如下步骤来配置限制。
```

File: content/zh/news/security/istio-security-2020-007/index.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+* 对于 1.5.x 部署： 请升级至  [Istio 1.5.7](/news/releases/1.5.x/announcing-1.5.7) 或更高的版本。
+* 对于 1.6.x 部署： 请升级至  [Istio 1.6.4](/news/releases/1.6.x/announcing-1.6.4) 或更高的版本。
+
+{{< warning >}}
+您必须采取以下附加步骤来防范 CVE-2020-8663。
+{{< /warning >}}
+
+
+通过添加[下游连接](https://www.envoyproxy.io/docs/envoy/v1.14.3/configuration/operations/overload_manager/overload_manager#limiting-active-connections)的配置限制，在 Envoy 解决 CVE-2020-8663。 必须配置此限制以防范此漏洞。 在 Ingress 网关上执行如下步骤来配置限制。 
+
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-007/index.md:45]:
```suggestion
通过添加[下游连接](https://www.envoyproxy.io/docs/envoy/v1.14.3/configuration/operations/overload_manager/overload_manager#limiting-active-connections)的配置限制，在 Envoy 解决 CVE-2020-8663。 必须配置此限制以防范此漏洞。在 Ingress 网关上执行如下步骤来配置限制。

```

File: content/zh/news/security/istio-security-2020-004/index.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+此外，此版本中还修复了另一个CVE，如下所述：
+[Kiali 安全告](https://kiali.io/news/security-bulletins/kiali-security-001/).
+
+## 检测{#detection}
+
+您的安装在以下配置中容易受到攻击：
+
+* Kiali 版本为1.15或更早版本。
+* Kiali 登录令牌和签名密钥未设置。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-004/index.md:30]:
```suggestion
* Kiali 登录令牌和签名密钥未设置。
```

File: content/zh/news/security/istio-security-2020-004/index.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+* Kiali 版本为1.15或更早版本。
+* Kiali 登录令牌和签名密钥未设置。 
+
+请运行以下命令，检查您的 Kiali 版本：
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system -l app=kiali -o yaml | grep image:
+{{< /text >}}
+
+请运行以下命令确定您的登录令牌是否未设置，并检查输出是否为空:
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-004/index.md:38]:
```suggestion
请运行以下命令确定您的登录令牌是否未设置，并检查输出是否为空：
```

File: content/zh/news/security/istio-security-2020-004/index.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+{{< text bash >}}
+$ kubectl get deploy kiali -n istio-system -o yaml | grep LOGIN_TOKEN_SIGNING_KEY
+{{< /text >}}
+
+请运行以下命令确定您的签名密钥是否未设置，并检查输出是否为空:
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-004/index.md:44]:
```suggestion
请运行以下命令确定您的签名密钥是否未设置，并检查输出是否为空：
```

File: content/zh/news/security/istio-security-2020-004/index.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+
+{{< text bash >}}
+$ kubectl get cm kiali -n istio-system -o yaml | grep signing_key
+{{< /text >}}
+
+## 防范{#mitigation}
+
+* 对于 1.4.x 部署： 请升级至 [Istio 1.4.7](/news/releases/1.4.x/announcing-1.4.7) 或更高的版本。
+* 对于 1.5.x 部署： 请升级至 [Istio 1.5.1](/news/releases/1.5.x/announcing-1.5.1) 或更高的版本。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-004/index.md:53]:
```suggestion
* 对于 1.4.x 部署： 请升级至 [Istio 1.4.7](/zh/news/releases/1.4.x/announcing-1.4.7) 或更高的版本。
* 对于 1.5.x 部署： 请升级至 [Istio 1.5.1](/zh/news/releases/1.5.x/announcing-1.5.1) 或更高的版本。
```

File: content/zh/news/security/istio-security-2020-006/index.md [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+      operation: MERGE
+      value:
+        typed_config:
+          "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
+          codec_type: HTTP1
+{{< /text >}}
+
+* 对于 Istio 1.4.x 部署: 请升级至 [Istio 1.4.10](/news/releases/1.4.x/announcing-1.4.10) 或更高的版本。
+* 对于 Istio 1.5.x 部署: 请升级至 [Istio 1.5.5](/news/releases/1.5.x/announcing-1.5.5) 或更高的版本。
+* 对于 Istio 1.6.x 部署: 请升级至 [Istio 1.6.2](/news/releases/1.6.x/announcing-1.6.2) 或更高的版本。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-006/index.md:55]:
```suggestion
* 对于 Istio 1.4.x 部署: 请升级至 [Istio 1.4.10](/zh/news/releases/1.4.x/announcing-1.4.10) 或更高的版本。
* 对于 Istio 1.5.x 部署: 请升级至 [Istio 1.5.5](/zh/news/releases/1.5.x/announcing-1.5.5) 或更高的版本。
* 对于 Istio 1.6.x 部署: 请升级至 [Istio 1.6.2](/zh/news/releases/1.6.x/announcing-1.6.2) 或更高的版本。
```

File: content/zh/news/security/istio-security-2020-007/index.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+* __[CVE-2020-12605](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-12605)__:
+在处理特制 HTTP/1.1 的数据包时，攻击者可能会导致 Envoy 消耗过多的内存。
+    * CVSS Score: 7.0 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H&version=3.1)
+
+* __[CVE-2020-8663](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8663)__:
+当接受太多连接时，攻击者可能导致 Envoy 耗尽文件描述符。
+    * CVSS Score: 7.0 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H&version=3.1)
+
+* __[CVE-2020-12604](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-12604)__:
+在处理特制数据包时，攻击者可能会导致内存使用增加。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-007/index.md:31]:
```suggestion
* __[CVE-2020-12603](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-12603)__：
通过发送特制数据包，攻击者可能会导致 Envoy 在代理 HTTP/2 请求或响应时消耗过多的内存。
    * CVSS Score: 7.0 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H&version=3.1)

* __[CVE-2020-12605](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-12605)__：
在处理特制 HTTP/1.1 的数据包时，攻击者可能会导致 Envoy 消耗过多的内存。
    * CVSS Score: 7.0 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H&version=3.1)

* __[CVE-2020-8663](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8663)__：
当接受太多连接时，攻击者可能导致 Envoy 耗尽文件描述符。
    * CVSS Score: 7.0 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H&version=3.1)

* __[CVE-2020-12604](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-12604)__：
在处理特制数据包时，攻击者可能会导致内存使用增加。
```

File: content/zh/news/security/istio-security-2020-007/index.md [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+    * CVSS Score: 5.3 [AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H&version=3.1)
+
+## 防范{#mitigation}
+
+* 对于 1.5.x 部署： 请升级至  [Istio 1.5.7](/news/releases/1.5.x/announcing-1.5.7) 或更高的版本。
+* 对于 1.6.x 部署： 请升级至  [Istio 1.6.4](/news/releases/1.6.x/announcing-1.6.4) 或更高的版本。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-007/index.md:37]:
```suggestion
* 对于 1.5.x 部署： 请升级至  [Istio 1.5.7](/zh/news/releases/1.5.x/announcing-1.5.7) 或更高的版本。
* 对于 1.6.x 部署： 请升级至  [Istio 1.6.4](/zh/news/releases/1.6.x/announcing-1.6.4) 或更高的版本。
```

File: content/zh/news/security/istio-security-2020-007/index.md [hunk 16]
```diff
@@ -0,0 +1,82 @@
...
+    $ kubectl -n istio-system apply -f custom-bootstrap-runtime.yaml
+    {{< /text >}}
+
+1. 使用以上配置来修补 Ingress 网关。 下载 [gateway-patch.yaml](/news/security/istio-security-2020-007/gateway-patch.yaml) 文件并使用如下命令。
+
+    {{< text bash >}}
+    $ kubectl --namespace istio-system patch deployment istio-ingressgateway --patch "$(cat gateway-patch.yaml)"
+    {{< /text >}}
+
+1. 确认新限制已经配置.
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-007/index.md:58]:
```suggestion
1. 确认新限制已经配置。
```

File: content/zh/news/security/istio-security-2020-008/index.md (hunk 4 above)
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-008/index.md:26]:
```suggestion
* 通过 [Envoy 筛选器](/zh/docs/reference/config/networking/envoy-filter/)直接使用 Envoy 的`verify_subject_alt_name`和`match_subject_alt_names`配置。

* 使用Istio 的[ `subjectAltNames` 字段在目标规则与客户端 TLS 设置](/zh/docs/reference/config/networking/destination-rule/#ClientTLSSettings).  具有包含 `nested.subdomain.example.com` 的 `subjectAltNames` 字段的目标规则错误地接受来自主机名称（SAN）为 `* .example.com` 的上游层证书。 相反，应该存在一个 `*.subdomain.example.com` 或 `nested.subdomain.example.com` 的 SAN。
```

File: content/zh/news/security/istio-security-2020-008/index.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+
+* 使用 Istio 的 [服务条目中的 `subjectAltNames`](/docs/reference/config/networking/service-entry/)。 带有 ` subjectAltNames` 字段的值类似于 `nested.subdomain.example.com` 的服务条目错误地接受来自 SAN 为 `*.example.com` 的上游层证书。
+
+Istio CA（以前称为 Citadel）不使用 DNS 通配符 SAN 颁发证书。 该漏洞仅影响验证外部颁发证书的配置。 
+
+## 防范{#mitigation}
+
+* 对于 1.5.x deployments: 部署： 请升级至 [Istio 1.5.8](/news/releases/1.5.x/announcing-1.5.8) 或更高的版本。
+* 对于 1.6.x deployments: 部署： 请升级至 [Istio 1.6.5](/news/releases/1.6.x/announcing-1.6.5) 或更高的版本。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-008/index.md:35]:
```suggestion
* 对于 1.5.x deployments: 部署： 请升级至 [Istio 1.5.8](/zh/news/releases/1.5.x/announcing-1.5.8) 或更高的版本。
* 对于 1.6.x deployments: 部署： 请升级至 [Istio 1.6.5](/zh/news/releases/1.6.x/announcing-1.6.5) 或更高的版本。
```

File: content/zh/news/security/istio-security-2020-009/index.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+stringMatch:
+  regex: spiffe://.*/ns/ns1/sa/foo
+{{< /text >}}
+
+`AuthorizationPolicy` 中的前缀和精确匹配不受影响，它们中的 `ALLOW` 操作也一样；HTTP 也不受影响。
+
+## 防范{#mitigation}
+
+* 对于 Istio 1.5.x 部署： 请升级至 [Istio 1.5.9](/news/releases/1.5.x/announcing-1.5.8) 或是更高的版本。
+* 对于 Istio 1.6.x 部署： 请升级至 [Istio 1.6.8](/news/releases/1.6.x/announcing-1.6.8) 或是更高的版本。
```
Comment (orangegzx) [inline content/zh/news/security/istio-security-2020-009/index.md:58]:
```suggestion
* 对于 Istio 1.5.x 部署： 请升级至 [Istio 1.5.9](/zh/news/releases/1.5.x/announcing-1.5.8) 或是更高的版本。
* 对于 Istio 1.6.x 部署： 请升级至 [Istio 1.6.8](/zh/news/releases/1.6.x/announcing-1.6.8) 或是更高的版本。
```

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**